src/lib/storage/chunk.hpp

```diff
@@ -108,6 +108,12 @@ class Chunk : private Noncopyable {
   Chunk(Chunk&&) = default;
   Chunk& operator=(Chunk&&) = default;
 
+  // returns id of the chunk
```

Comment:
Could you please remind me what this is needed for?

Reply:
This is faster than asking the table for the chunk id every time. Used for example in insert, export, and print.